docs/Change_Log.md

```diff
@@ -1,5 +1,13 @@
 # Change Log
 
+## Forum Channel Layouts
+
+#### Nov 28, 2022
+
+Forum channels now have a new media-focused gallery view option. The default view can be set with the new `default_forum_layout` field in the [channel object](#DOCS_RESOURCES_CHANNEL). Setting `default_forum_layout` requires the `MANAGE_CHANNELS` permission.
+
+If `default_forum_layout` hasn't been set, its value will be `0` (`DEFAULT`).
```

Comment:
How come this defaults to 0 and `default_sort_order` defaults to `null`? Seems strange.